<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YI-Hack-Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/resources.qrc">
    <normaloff>:/icons/icons/logo.svg</normaloff>:/icons/icons/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="editCameraBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../data/resources.qrc">
            <normaloff>:/icons/icons/edit.svg</normaloff>:/icons/icons/edit.svg</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playBtn">
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset resource="../data/resources.qrc">
            <normaloff>:/icons/icons/play.svg</normaloff>:/icons/icons/play.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ftpBtn">
          <property name="text">
           <string>FTP</string>
          </property>
          <property name="icon">
           <iconset resource="../data/resources.qrc">
            <normaloff>:/icons/icons/ftp.svg</normaloff>:/icons/icons/ftp.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item row="0" column="0">
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAddCamera"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddCamera">
   <property name="text">
    <string>Add Camera</string>
   </property>
  </action>
  <action name="actionDeleteCamera">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../data/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tableView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>play(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCameraBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>editSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddCamera</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindow</receiver>
   <slot>tableViewContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeleteCamera</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ftpBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFtp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editCamera(QModelIndex)</slot>
  <slot>play(QModelIndex)</slot>
  <slot>playSelected()</slot>
  <slot>editSelected()</slot>
  <slot>addCamera()</slot>
  <slot>tableViewContextMenu(QPoint)</slot>
  <slot>deleteCamera()</slot>
  <slot>openFtp()</slot>
 </slots>
</ui>
